src/vs/platform/storage/common/storage.ts

```diff
@@ -231,6 +236,19 @@ interface IKeyTargets {
 
 export interface IStorageServiceOptions {
 	flushInterval: number;
+	donotMarkPerf?: boolean;
```

Comment:
readonly?

And better `doNotMarkPerf`? seems like a hack though, maybe we dont need it?